opm/simulators/timestepping/AdaptiveTimeStepping.cpp

```diff
@@ -33,7 +33,7 @@ void logTimer(const AdaptiveSimulatorTimer& substepTimer)
 }
 
 std::set<std::string>
-consistentlyFailingWells(const std::vector<StepReport>& sr)
+consistentlyFailingWells(const std::vector<StepReport>& sr, bool checkRepeatedFailures)
```

Comment:
I misunderstood the intention at first, maybe call the argument `requireRepeatedFailuresToFlag`? Also, make it const.